Sources/BFloat16/BFloat16.swift

```diff
@@ -1,494 +1,59 @@
 //
-//  BFloat.swift
-//  BFloat
+//  BFloat16.swift
+//  BFloat16
 //
 //  Created by Ivar Flakstad on 22/02/2024.
 //
 
 import Swift
 import SwiftShims
 
+#if SWIFT_PACKAGE
+import bfloat16_c
+#endif
 
 @frozen
 public struct BFloat16 {
   @usableFromInline @inline(__always)
-  internal var _value: UInt16 = 0x0000
+  internal var _value: bf16_t;
+  
+  @_transparent
+  public init() {
+    _value = 0;
+  }
+  
+  @_transparent @inlinable @inline(__always)
+  init(_ value: bf16_t) {
+    _value = value
+  }
   
   @_transparent @inlinable @inline(__always)
   init(bitPattern: UInt16) {
-    _value = bitPattern
+    _value = bf16_from(bitPattern)
   }
   
-  @usableFromInline @inline(__always)
-  func toFloat() -> Float {
-    if _fastPath(self.bitPattern & 0x7FFF <= 0x7F80) {
-      return Float(bitPattern: UInt32(self.bitPattern) << 16)
-    } else {
-      return Float(bitPattern: (UInt32(self.bitPattern) | UInt32(0x0040)) << 16)
-    }
+  @inlinable public static var zero: BFloat16 {
+    BFloat16()
   }
   
-  @usableFromInline @inline(__always)
-  static func fromFloat(_ value: Float) -> BFloat16 {
-    let x: UInt32 = value.bitPattern;
-    if _fastPath(x & 0x7FFF_FFFF <= 0x7F80_0000) {
-      let round_bit = UInt32(0x0000_8000);
-      if (x & round_bit) != 0 && (x & (3 * round_bit - 1)) != 0 {
-        return BFloat16(bitPattern: UInt16(x >> 16) + 1)
-      } else {
-        return BFloat16(bitPattern: UInt16(x >> 16))
-      }
-    } else {
-      return BFloat16(bitPattern: UInt16((x >> 16) | 0x0040))
-    }
+  @inlinable public static var neg_zero: BFloat16 {
```

Comment:
It's probably better to use longer names for public properties, e.g. `negativeZero` and `negativeOne`.